### PR TITLE
creating BEM class generator, Header initial

### PR DESCRIPTION
**Pull request by donmikelee**
I have initialized the header component. Please look at the method for building class names for components. csApp is the main block for the entire application, where cs is an abbreviation of country-sercher - the name of the application.

Can such a method be in the app component or should it be separated as a separate file, e.g. CssHelper.ts

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+
+  return className
+}
+
 const App = () => {
+
+  const headerClass = generateBEMClassName( { block: 'csApp', element: 'Header' });
   return ( 
-    <div className="csApp"></div>
+    <div className="csApp">
```
Comment (AdamKniec) [inline src/App.tsx]: I don't like this name csApp.  Why not simply appWrapper or something like that?

File: src/App.tsx [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
+      <Header className={headerClass}>Dupa</Header>
```
Comment (AdamKniec) [inline src/App.tsx]: do not do this ever again. Good tip for you 👍 ;)

File: src/components/sections/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+interface HeaderProps {
+    className: string;
+    children: string;
+}
+
+const Header: React.FC<HeaderProps> = ({className, children}) => {
```
Comment (AdamKniec) [inline src/components/sections/Header.tsx]: Why do you expect the header to take children?

File: src/App.tsx (hunk 1 above)
Comment (donmikelee) [reply src/App.tsx]: Why you don't like name csApp?

File: src/App.tsx (hunk 2 above)
Comment (donmikelee) [reply src/App.tsx]: Jokes aside

File: src/components/sections/Header.tsx (hunk 3 above)
Comment (donmikelee) [reply src/components/sections/Header.tsx]: I don't, just rendered a chilren to see something on my localhost.

File: src/App.tsx (hunk 1 above)
Comment (donmikelee) [reply src/App.tsx]: What about csAppWrapper?

Comment (AdamKniec) [reply src/App.tsx]: it's because noone will probably know what cs stands for. appWrapper should be enough 👍

**Pull request merged**